tests/e2e_tests/Dockerfile.gui

```diff
@@ -0,0 +1,23 @@
+FROM python:2
+WORKDIR /usr/src/app
+ENV CHROME_BIN="/usr/bin/google-chrome"
+ENV DISPLAY=":99.0"
+RUN apt-get -q update && apt-get -yq install chromium=73.0.3683.75-1~deb9u1
+RUN wget -q "https://chromedriver.storage.googleapis.com/73.0.3683.68/chromedriver_linux64.zip" -O /tmp/chromedriver.zip \
+  && unzip /tmp/chromedriver.zip -d /usr/bin/ \
+  && rm /tmp/chromedriver.zip
+RUN pip install -U --user paramiko 'fabric<2.0' PyYAML pytest requests pytest-xdist filelock selenium psutil
```

Comment:
I don't think we are using Fabric. If I am not mistaken, it should be dropped (which is probably the reason to use Python 2).

Reply:
fabric is unfortunately required in one of the helper function - thus the python 2 requirement

Reply:
:+1: